as2/server/player.md

```diff
@@ -0,0 +1,84 @@
+# Player
+##### Messages pertaining to the player actions.
+---
+### Send Frame
+Sent to all penguins in a room to update a player frame. `sf`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|penguinID|Unsigned integer|No|
+|Frame|Unsigned integer|No|
+
+##### Example
+`%xt%sf%0%123%5%`
+
+### Send Move
+Sent to all penguins in a room to update a player location. `sp`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|penguinID|Unsigned integer|No|
+|X Postion|Unsigned integer|No|
+|Y Postion|Unsigned integer|No|
+
+##### Example
+`%xt%sp%0%123%0%0%`
+
+### Send Throw Snowball
+Sent to all penguins in a room to show a snowball thrown by a penguin. `sb`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|penguinID|Unsigned integer|No|
+|X Postion|Unsigned integer|No|
+|Y Postion|Unsigned integer|No|
+
+##### Example
+`%xt%sb%0%123%0%0%`
+
+
+### Send Emote
+Sent to all penguins in a room to show an emote sent by a penguin. `se`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|penguinID|Unsigned integer|No|
+|Emote ID|Unsigned integer|No|
+
+##### Example
+`%xt%se%0%123%3%`
+
+### Send Joke
+Sent to all penguins in a room to show a joke sent by a penguin. `sj`
```

Comment:
"joke sent" should be "joke said" to better describe the client's visual behaviour.